launch/jetson.launch

```diff
@@ -1,10 +1,25 @@
+<!-- 
+	This launch file should be run on the Jetson in order to start our drive, control, and auton software
+	on the physical rover.
+ -->
 <launch>
-  <!--Node for teleop control callbacks-->
-  <rosparam command="load" file="$(find mrover)/config/teleop.yaml" />
-  <node name="jetson_teleop" pkg="mrover" type="jetson_teleop.py"/>
-  <!--
-    Run the drive program
-  -->
-  <node name="odrive_control" pkg="mrover" type="odrive_control.py" output="screen"></node>
-  <rosparam command="load" file="$(find mrover)/config/esw.yaml" />
+	<!-- launch rover core nodes -->
+	<include file="$(find mrover)/launch/auton.launch" />
+
+	<!-- params for hardware interfaces including IMU, GPS, and odrives -->
+	<rosparam command="load" file="$(find mrover)/config/esw.yaml" />
+
+	<!-- IMU and GPS driver nodes -->
+	<node name="imu_driver" pkg="mrover" type="imu_driver.py" output="screen" />
+	<node name="gps_driver" pkg="nmea_navsat_driver" type="nmea_serial_driver" />
+
+	<!-- teleop node for receiving commands from the gui -->
+	<rosparam command="load" file="$(find mrover)/config/teleop.yaml" />
+	<node name="jetson_teleop" pkg="mrover" type="jetson_teleop.py" />
+
+	<!-- odrive controller node -->
+	<node name="odrive_control" pkg="mrover" type="odrive_control.py" output="screen" />
+
+	<!-- launch auton -->
+	<include file="$(find mrover)/launch/auton.launch" />
```

Comment:
written twice

Reply:
fixed, the first one was supposed to be rover_core